app/views/admin/organizations/_loans.html.slim

```diff
@@ -1,4 +1,11 @@
-h1 = t('organization.active_loans')
+/ JE Todo: Confirm how these styles should be factored
+div style="margin-bottom: 15px"
+  h1 style="display: inline"
+    = t('organization.active_loans')
+  span style="margin-left: 10px"
+    = link_to t(".view_all_loans"), admin_loans_path({'loans[f][organizations.name][]' => @org.name})
+  span style="float: right"
+    = link_to t("loan.shared.new"), new_admin_loan_path({organization_id: @org.id}), class: 'btn btn-default new-button'
```

Comment:
I would make a style inside the .block style in general called e.g. .header-with-links. Then you can put your margin in there, and then descendant h1's can be made inline-block, any link can have a margin-left (no need for a wrapping span, links should also be made inline-block), and any button can be floated right. That should get rid of all the inline stuff.